### PR TITLE
Updated reference directory for lesson 1.4 to match lesson on wiki.

### PR DESCRIPTION
**Pull request by Jmeyer1292**
Resolves an issue related to a disparity between autogenerated names in CMakeLists.txt and the names on the industrial training wiki for ex 1.4.

### CODE REVIEW

File: training/ref/1.4/src/lesson_simple/package.xml [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
   <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
-  <maintainer email="jane.doe@example.com">Jane Doe</maintainer>
+  <maintainer email="ros-industrial@todo.todo">ros-industrial</maintainer>
 
 
   <!-- One license tag required, multiple allowed, one license per tag -->
   <!-- Commonly used license strings: -->
   <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
-  <license>BSD</license>
+  <license>TODO</license>
```
Comment (gomezc) [inline training/ref/1.4/src/lesson_simple/package.xml]: I think we generally to update this to the BSD license.

**Comment by gomezc** [comment]
@Jmeyer1292 Just the one small comment on the package.xml

File: training/ref/1.4/src/lesson_simple/package.xml (hunk 1 above)
Comment (Jmeyer1292) [inline training/ref/1.4/src/lesson_simple/package.xml]: My bad, +1. I'll fix that.

**Comment by Jmeyer1292** [comment]
@gomezc I updated the license and maintainer tags to their previous values.

**Comment by gomezc** [comment]
+1

**Pull request merged**